src/domain/occurrences/occurrencesTable/OccurrencesTable.tsx

```diff
@@ -148,6 +156,7 @@ const OccurrencesTable: React.FC<Props> = ({
             <EnrolmentsBadge
               acceptedSeatsCount={row.seatsApproved}
```

Comment:
There seems to be synonyms: `acceptedSeats === approvedSeats` Which lead to confusion. Synonyms should be avoided in codebase. I'm not sure though is it too big task to rename one to another in this pr? But it would make code cleaner at least to me :)

Reply:
renamed to approvedSeats